Library/Homebrew/brew.sh

```diff
@@ -1,5 +1,8 @@
-HOMEBREW_VERSION="$(git -C "$HOMEBREW_REPOSITORY" describe --tags --dirty 2>/dev/null)"
-HOMEBREW_USER_AGENT_VERSION="$HOMEBREW_VERSION"
+if [[ $(git -C "$HOMEBREW_REPOSITORY" status) ]]
```

Comment:
Running <kbd>git status</kbd> seems a little overkill. Wouldn't it be enough to check if `"$HOMEBREW_REPOSITORY"/.git` exists?